### PR TITLE
Fix NW bugs

### PR DESCRIPTION
**Pull request by skyyt15**
## Description
Fix some bugs from NW trello

**What is the current behavior?**
  - https://git.scpslgame.com/northwood-qa/scpsl-bug-reporting/-/issues/230
  - https://git.scpslgame.com/northwood-qa/scpsl-bug-reporting/-/issues/143

**What is the new behavior?** Fix bugs

**Does this PR introduce a breaking change?** No

**Other information**:

<br />

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [x] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [x] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

**Review by louis1706 — APPROVED** [review]

File: EXILED/Exiled.Events/Patches/Generic/Scp173FirstKillPatch.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
+
+            Label continueLabel = generator.DefineLabel();
+
+            int index = newInstructions.FindLastIndex(x => x.opcode == OpCodes.Ldc_I4_0);
+            newInstructions[index].WithLabels(continueLabel);
+
+            // TODO: NRE ON LEFT CLICK (KICK PLAYER) - DO NOT PUSH
```
Comment (louis1706) [inline EXILED/Exiled.Events/Patches/Generic/Scp173FirstKillPatch.cs:36]: need to be finish

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.Events/Patches/Generic/Scp173FirstKillPatch.cs (hunk 1 above)
Comment (skyyt15) [reply EXILED/Exiled.Events/Patches/Generic/Scp173FirstKillPatch.cs:36]: yes i know

Comment (skyyt15) [reply EXILED/Exiled.Events/Patches/Generic/Scp173FirstKillPatch.cs]: fixed

File: EXILED/Exiled.Events/Patches/Fixes/SlownessFix.cs [hunk 2]
```diff
@@ -62,4 +63,40 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
...
+                new(OpCodes.Starg_S, 1),
+            });
+
+            for (int z = 0; z < newInstructions.Count; z++)
+                yield return newInstructions[z];
+
+            ListPool<CodeInstruction>.Pool.Return(newInstructions);
+        }
+    }
+}
```
Comment (louis1706) [inline EXILED/Exiled.Events/Patches/Fixes/SlownessFix.cs:102]:
```suggestion
}
```

**Review by louis1706 — APPROVED** [review]
If everything work it's fine

**Review by Vladislav-CS — CHANGES_REQUESTED** [review]
PRs should target dev branch, not master

**Review by Misfiy — APPROVED** [review]

**Pull request merged**